Automattic/jetpack
# VideoPress: implement "Privacy and rating" panel to v6

This PR creates and adds the new `Privacy and rating` panel, with the `Privacy` control.
The implementation registers a new `privacySetting` attribute used to render the UI smoothly but also keeps the attribute values in sync with the VideoPress video data by using the `useSyncMedia()` hook.



Fixes https://github.com/Automattic/jetpack/issues/27101

#### Changes proposed in this Pull Request:

* VideoPress: implement Privacy and rating panel

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Create/edit a VideoPress video block
* Open the block sidebar
* Confirm you see the `Privacy and rating` panel
* Change the privacy value
* hard refresh
* confirm the data is persistent
* Additionally, you may like to take a look at the network tab to see how the data is synced when the post saves


https://user-images.githubusercontent.com/77539/201758884-1d65c21c-b360-4deb-83a6-cb3acbd3c6c1.mov
